### ISSUE TITLE
KSQL DB Java Client Configuration Document requires update

### ISSUE THREAD

**Issue by pallavnagar02** [confluentinc/ksql#9245]
Please update this Document with required configurations to setup KSQL DB Java Client using Confluent Platform and Java Configurations. 

https://docs.ksqldb.io/en/latest/developer-guide/ksqldb-clients/java-client/#connect-to-a-confluent-cloud-ksqldb-cluster

I have also raised this Issue over Confluent Community Slack channel #ksqldb dated June 24th.

Please assist by improvements or updating the documentation with required configuration and code updates on Git too.

Thanks

**Comment by jnh5y** [comment]
@suhas-satish Yes.  If we shade the dependencies, we could document how to use the new shaded dependency at that time.  I can take it (if no one else comes to mind).

**Comment by JimGalasyn** [comment]
@jnh5y Great, if you can do that, I can help to get it published.

**Comment by jnh5y** [comment]
The merged PR should close this out once the docs are merged in 0.29.
